week2/4_Lists.ipynb

```diff
@@ -186,7 +186,7 @@
     "מה התרחש בקוד?<br>\n",
     "התחלנו את הגדרת הרשימה באמצעות התו <code dir=\"ltr\" style=\"direction: ltr;\">[</code>.<br>\n",
     "מיד אחרי התו הזה דאגנו להכניס איברים לרשימה לפי הסדר הרצוי, כאשר כל איבר מופרד ממשניהו בתו פסיק (<code>,</code>).<br>\n",
-    "במקרה שלנו, כל איבר הוא מחרוזת המייצגת ראש ממשלה, והם <mark>מסודרים ברשימה</mark> לפי סדר כהונתם.<br>\n",
+    "במקרה שלנו, כל איבר הוא מחרוזת המייצגת ראש ממשלה ודאגנו להגדיר את הרשימה כך שתכיל את הערכים (ראשי ממשלה) <mark>לפי סדר</mark> כהונתם.<br>\n",
```

Comment:
This is just more verbose. What did you try to accomplish here? :)

Reply:
My thought was to explain that we explicitly declared the list in the right order.
It might be too explicit, if so I'll change it back :)

Reply:
Maybe
"ודאגנו להכניס את ראשי הממשלה לרשימה \<mark>לפי סדר\</mark> כהונתם"?

Reply:
, before ודאגנו